core/mm/core_mmu.c

```diff
@@ -642,6 +643,13 @@ uint32_t core_mmu_type_to_attr(enum teecore_memtypes t)
 	case MEM_AREA_NSEC_SHM:
 		return attr | TEE_MATTR_PRW | cached;
 	case MEM_AREA_EXT_DT:
+	/*
+	 * If CFG_MAP_EXT_DT_SECURE is enabled map the external device tree as
+	 * secure non-cached memory, otherwise, fall back to non-secure mapping.
+	 */
+#ifdef CFG_MAP_EXT_DT_SECURE
```

Comment:
Please use `if (IS_ENABLED(CFG_MAP_EXT_DT_SECURE))` instead.

Reply:
Done, also added fallthrough; to suppress compiler warning.